src/com/hotmail/kalebmarc/textfighter/main/Enemy.java

```diff
@@ -109,6 +110,11 @@ void dealDamage() {
         Health.takeDamage(damage);
     }
 
+    void returnDamage(int damage) {
+        damageDealt = damage;

```

Comment:
although this works, its common to refer to non-static class variables with the `this` keyword. 
```suggestion
        this.damageDealt = damage;
```

Reply:
Does it have any particular reason, or does it make the code just look just a bit cleaner?

Reply:
@R1ndT the only benefit, in this case, is it just makes it obvious that the variable is a class property, and I don't have to scroll up to see what it's coming from

In general, though, it can prevent accidentally accessing the wrong variable (maybe there's a variable with the same name in scope).

This is another case where it's not really a big deal in smaller projects, but as the project grows (or you work on very large projects), it is helpful to be very explicit about every action because the codebase can become confusing with size